src/MlCli/JobApiCall/Job.cs

```diff
@@ -107,6 +108,11 @@ public async Task ApiCallAsync(Callapi inputTask)
                             numberKo += 1;
                             _logger.LogWarning("number KO: " + numberKo + "/" + (indexFile + 1));
                         }
+                        else if (task.Result == "Exception")
```

Comment:
Tu as une manière plus propre de faire cela => https://stackoverflow.com/questions/32273165/task-faulted-and-task-exception